### PR TITLE
Ux/2292 add suite auth issuer to select in the technical user creation dialogue

### PR DESCRIPTION
**Pull request by mena-bosch**

### CODE REVIEW

File: repository/repository-core/src/main/java/org/eclipse/vorto/repository/web/oauth/OAuthProvider.java [hunk 1]
```diff
@@ -24,10 +30,52 @@
...
+
+  public static OAuthProvider from(IOAuthProvider source) {
+    OAuthProvider result = new OAuthProvider();
+    result.setId(source.getId());
+    result.setLabel(source.getLabel());
+    if (source.getWebflowConfiguration().isPresent()) {
+      IOAuthFlowConfiguration config = source.getWebflowConfiguration().get();
+      result.setLogoHref(config.getLogoHref());
+      result.setLoginUrl(config.getFilterProcessingUrl());
+      // TODO this is temporary and ugly: assigning the Bosch ID icon to the non-webflow providers
```
Comment (kolotu) [inline repository/repository-core/src/main/java/org/eclipse/vorto/repository/web/oauth/OAuthProvider.java]: TODO? What's left to do here?

Comment (mena-bosch) [reply repository/repository-core/src/main/java/org/eclipse/vorto/repository/web/oauth/OAuthProvider.java]: > TODO? What's left to do here?

Resolved conversation by mistake. I think the TODO should stay until we figure out what exactly we want to do with the icon.

Comment (kolotu) [reply repository/repository-core/src/main/java/org/eclipse/vorto/repository/web/oauth/OAuthProvider.java]: tbh I don't think we will ever do smth about the icon.

Comment (mena-bosch) [reply repository/repository-core/src/main/java/org/eclipse/vorto/repository/web/oauth/OAuthProvider.java]: @kolotu ye of little faith. Allright I'll bury the TODO then

Comment (kolotu) [reply repository/repository-core/src/main/java/org/eclipse/vorto/repository/web/oauth/OAuthProvider.java]: @mena-bosch thanks :)

**Comment by mena-bosch** [comment]
@kolotu do you want the honors?

**Pull request merged**